### ISSUE TITLE
hi,I meet with a question that train's miou is far heighter than test's,can you give me some advice?

### ISSUE THREAD

**Issue by Gayou333** [CoinCheung/BiSeNet#303]

**Comment by CoinCheung** [comment]
In principle, this means overfitting. Maybe you can add more data to train your model, and you can train for less iterations. I do not think a large gap between the performance on training and test set is always a bad thing. If the performance on the test set is good, then we would still accept the large performance gap.

**Comment by CoinCheung** [comment]
I am closing this since question is answered.
